Salmon/0.13.1/Index.nf

```diff
@@ -0,0 +1,21 @@
+process Index {
+    tag {"Salmon Index ${transcripts_fasta.baseName}"}
+    label 'Salmon_0_13_1'
+    label 'Salmon_0_13_1_Index'
+    clusterOptions = workflow.profile == "sge" ? "-l h_vmem=${params.mem}" : ""
+    container = 'quay.io/biocontainers/salmon:0.13.1--h86b0361_0'
+    shell = ['/bin/bash', '-euo', 'pipefail']
+    
+    input:
+    file(transcripts_fasta)
+    
+   
+    output:
+    file("${transcripts_fasta.baseName}/")
+
+    shell:
```

Comment:
Wat is het verschil tussen shell en script?

Reply:
Denk er zijn verschillen wat het escapen van environment variables, new line characters etc. betreft. Functioneel lijjkt het niets uit te maken. Ik heb het aangepast.